zib/StructureDefinitions/ext-TimeInterval.Period.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-TimeInterval.Period" />
+  <text>
+    <status value="empty"/>
+    <div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div>
+  </text>
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval.Period" />
+  <name value="ExtTimeIntervalPeriod" />
+  <title value="ext TimeInterval.Period" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An extension to provide all concepts of zib TimeInterval, to be used in resources where none of these concepts are natively supported." />
+  <purpose value="This extension is one of the three StructureDefinitions that can be used to represent the Dutch (partial) [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) TimeInterval v1.3 (2024)](https://zibs.nl/wiki/TimeInterval-v1.3(2024EN)). &#xD;&#xA;&#xD;&#xA;This zib is represented in FHIR by a combination of two datatypes: _Period_ and _Duration_. To support this zib across different FHIR resource types, three related StructureDefinitions are provided:&#xD;&#xA;&#xD;&#xA;* [ext-TimeInterval.Duration](http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval.Duration): extension to provide the _Duration_ datatype to resource types where no element with this datatype isn't normally available. This extension should not be used in isolation, but only via one of the two resources described below!&#xD;&#xA;* [zib-TimeInterval](http://nictiz.nl/fhir/StructureDefinition/zib-TimeInterval): profile on the _Period_ datatype that defines mappings to zib TimeInterval and brings in the ext-TimeInterval.Duration extension mentioned above. For resource types that natively support an element of the _Period_ datatype but not of the _Duration_ datatype, this profile should be used.&#xD;&#xA;* [ext-TimeInterval.Period](http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval.Period) (this extension): extension to provide the _Period_ datatype plus the _Duration_ extension mentioned above, with mappings to zib TimeInterval. For resource types that support neither elements of datatype _Period_ nor _Duration_, this extension should be used. *Note*: In hindsight the name of this extension is confusing, 'ext-TimeInterval' would have been better suited.&#xD;&#xA;* *Note*: when the resource type natively supports elements of datatype _Period_ and _Duration_, the zib should be mapped onto these elements directly." />
```

Comment:
I don't understand these two bullets:

* [zib-TimeInterval](http://nictiz.nl/fhir/StructureDefinition/zib-TimeInterval): profile on the _Period_ datatype that defines mappings to zib TimeInterval and brings in the ext-TimeInterval.Duration extension mentioned above. For resource types that natively support an element of the _Period_ datatype but not of the _Duration_ datatype, this profile should be used.
* [ext-TimeInterval.Period](http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval.Period) (this extension): extension to provide the _Period_ datatype plus the _Duration_ extension mentioned above, with mappings to zib TimeInterval. For resource types that support neither elements of datatype _Period_ nor _Duration_, this extension should be used. *Note*: In hindsight the name of this extension is confusing, 'ext-TimeInterval' would have been better suited.

There is only one profile on the Period datatype? [zib-TimeInterval](http://nictiz.nl/fhir/StructureDefinition/zib-TimeInterval) does not exists.

Reply:
Sorry, I read this wrong. This is no issue.